chef/cookbooks/horizon/recipes/monasca_ui.rb

```diff
@@ -0,0 +1,38 @@
+# Copyright 2016 SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+monasca_servers = search(:node, "roles:monasca-server")
+monasca_server = monasca_servers[0]
```

Comment:
I'm sorry, but I simply can't make much sense from the github review process...

It seems to me that your comments refer to some older version of this PR - I have already updated the code, so that right now grafana is deployed on the dashboard node(s). And, as a result, there is no more need to have any URLs to grafana